style.css

```diff
@@ -0,0 +1,62 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+#img1 {
+  justify-content: flex-start;
+}
+
+.ic_icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex: 0;
+  padding: 8%;
+}
+
+#btn {
+  justify-content: space-between;
+  border-radius: 8%;
+  background: #1a2236;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.group48 {
+  display: flex;
+  flex-direction: column;
+}
+
+#title {
+  font-family: 'Courier New', Courier, monospace; /* Playfair Displa; */
```

Comment:
- [ ] Kindly use the same font family as the Figma design (Playfair Display) for the main title to follow the project requirement.